apps/web/src/services/dashboards/widgets/asset-widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue

```diff
@@ -129,18 +151,67 @@ const { colorSet } = useWidgetColorSet({
     dataSize: computed(() => 1),
 });
 const state = reactive({
-    ...toRefs(useWidgetState<Data>(props)),
-    chartData: computed<XYChartData[]>(() => state.data),
+    ...toRefs(useWidgetState<Data[]>(props)),
+    // TODO: set real data
+    chartData: computed<XYChartData[]>(() => ([
+        { date: '2022-06', value: random(500, 1200) },
+        { date: '2022-07', value: random(500, 1200) },
+        {
+            date: '2022-08',
+            value: random(500, 1200),
+        },
+        {
+            date: '2022-09',
+            value: random(500, 1200),
+        },
+        {
+            date: '2022-10',
+            value: random(500, 1200),
+        },
+        {
+            date: '2022-11',
+            value: random(500, 1200),
+        },
+        {
+            date: '2022-12',
+            value: random(500, 1200),
+        },
+        {
+            date: '2023-01',
+            value: random(500, 1200),
+        },
+        {
+            date: '2023-02',
+            value: random(500, 1200),
+        },
+        {
+            date: '2023-03',
+            value: random(500, 1200),
+        },
+        {
+            date: '2023-04',
+            value: random(500, 1200),
+        },
+        {
+            date: '2023-05',
+            value: random(500, 1200),
+        },
+    ])),
     dateRange: computed<DateRange>(() => {
         const end = dayjs.utc(state.settings?.date_range?.end).format(DATE_FORMAT);
         const start = dayjs.utc(end).subtract(11, 'month').format(DATE_FORMAT);
         return { start, end };
     }),
-    severityData: computed(() => {
-        const results: any[] = []; // TODO: fix type after get real data
+    dateRangeDiff: computed<number>(() => {
+        const end = dayjs.utc(state.settings?.date_range?.end);
+        const start = dayjs.utc(state.settings?.date_range?.start);
+        return end.diff(start, 'month');
+    }),
```

Comment:
description: `dateRangeDiff` is for this section

![스크린샷 2023-06-08 오후 2 48 22](https://github.com/cloudforet-io/console/assets/18563857/5fe914a2-b6c6-4032-919d-bc1de45e7a93)